### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by dharmendra-lingaiah**
Made it easier to read & understand : Minor grammar & punctuation changes

### CODE REVIEW

**Comment by dampee** [comment]
@marcemarc Can you help for the first line?  The others are no-brainers and can be merged.

**Comment by dharmendra-lingaiah** [comment]
Each section of the Umbraco backoffice has its own set of default dashboards
to
"Each section in Umbraco backoffice has a set of default dashboards"

File: Extending/Dashboards/index.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 Each section of the Umbraco backoffice has its own set of default dashboards.
 
-The dashboard area of Umbraco is used to display the current editor for whichever item is selected in the tree, but when no items are selected, for example when the section first loads, that's when the default set of dashboards are displayed for that section, arranged over multiple tabs.
+The dashboard area of Umbraco is used to display the current editor for an item selected in the tree, but if no items are selected, for example when the section first loads, that's when the default set of dashboards are displayed for that section, arranged over multiple tabs.
```
Comment (marcemarc) [inline Extending/Dashboards/index.md]: How about:

The dashboard area of Umbraco is used to display an 'editor' for the selected item in the tree. If no item is selected, for example when the section 'first loads', then the default set of section dashboards are displayed in the dashboard area, arranged over multiple tabs.

Comment (dharmendra-lingaiah) [reply Extending/Dashboards/index.md]: ,Sounds great 👍

Comment (dharmendra-lingaiah) [reply Extending/Dashboards/index.md]: @marcemarc @dampee 
Updated  as  per review comments

**Comment by jmayntzhusen** [comment]
Hey @dharmendra-lingaiah 

Thanks for the fixes, will merge now 😊

**Comment by nul800sebastiaan** [comment]
Hi there @dharmendra-lingaiah,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**